www/index.html.spt

```diff
@@ -94,7 +94,7 @@ recent = query_cache.one("""
 
         % if user.ANON
 
-            <p>{{ _("Are you a creator of commons? Do you make free art, spread free knowledge, write free software?") }}</p>
+            <p>{{ _("Are you a creator of commons? Do you make free art, spread free knowledge, write libre software?") }}</p>
```

Comment:
Same as above. _Libera_pay, libre software

Reply:
_Libera_pay, libre art and libre knowledge? I don't see this as an improvement.

Reply:
So Liberapay is not a play on words alluding to libre software? Happy coincidence if not.